lib/testing/src/lib/process-services-cloud/pages/people-cloud-component.page.ts

```diff
@@ -81,6 +87,10 @@ export class PeopleCloudComponentPage {
         await BrowserVisibility.waitUntilElementIsNotVisible(assigneeRow);
     }
 
+    async getNoOfUsersDisplayed(): Promise<number> {
```

Comment:
same as before

Reply:
no means number

Reply:
let's avoid abbreviation when can create confusion

Reply:
So what name should i use?